src/components/NavBar/ChainSelector.tsx

```diff
@@ -76,7 +76,7 @@ export const ChainSelector = ({ leftAlign }: ChainSelectorProps) => {
       <Column paddingX="8">
         {NETWORK_SELECTOR_CHAINS.map((chainId: SupportedChainId) => (
           <ChainSelectorRow
-            disabled={isUniWallet && chainId === SupportedChainId.CELO}
+            disabled={isUniWallet && !UniWalletSupportedChains.includes(chainId)}
```

Comment:
I find it generally a bit confusing that this feature - displayed in the UI as a generic "not supported by your wallet" feature - is specific to Uniwallet. In that case, I'd rather see the wording "not supported by Uniwallet". 

Also, it would be great if this feature was expanded in the future into more generic abstraction, especially as we add WalletConnect multi-chain support where all chains become optional and when not present after connecting - it will mean that a give chain is unsupported.

I understand it's out of scope for this PR, so would rather have a follow-up ticket. If you agree, I would appreciate if you could create one!